src/test/unit_tests/test_nc_nsfs_cli.js

```diff
@@ -309,19 +293,19 @@ mocha.describe('manage_nsfs cli', function() {
             }
         });
 
-        mocha.it('cli bucket update bucket owner', async function() {
-            const action = nc_nsfs_manage_actions.UPDATE;
+        mocha.it('cli bucket update owner email', async function() {
```

Comment:
Why did you change it back to email?

Reply:
it came back during conflicts resolutions